src/lib/markdowndb.ts

```diff
@@ -191,3 +193,17 @@ export class MarkdownDB {
     this.db.destroy();
   }
 }
+
```

Comment:
Can we get function documentation for important public functions like that ... (jsdoc format would be nice).

Reply:
Okay, but I would like to create a new issue/pull request for that? I'd like to incorporate a few changes:
1. Include `strict: true`, as it is considered a best practice. This modification will necessitate some updates in the code but will substantially reduce the occurrence of bugs.
2. Initiate a discussion on how the library should behave on Windows versus macOS concerning `file_path`. Additionally, provide a fix for unit tests (as they currently fail on Windows due to path library differences)."
![image](https://github.com/datopian/markdowndb/assets/67981832/09122d92-d889-4fca-9a62-547832d470ad)

Reply:
OK.